### ISSUE TITLE
Fail to build front: apt install gettext - Release file is not valid yet

### ISSUE THREAD

**Issue by MaxenceG2M** [voyages-sncf-technologies/vboard#97]
Hi,

When I try to build front, I have an error on apt install gettext:
```
$ docker build -t vboard-front .
Sending build context to Docker daemon  2.799MB
Step 1/20 : FROM node:8-alpine
 ---> 2b8fcdc6230a
Step 2/20 : WORKDIR /usr/src/app
 ---> Using cache
 ---> 56b9122fb435
Step 3/20 : COPY .pleeeaserc .
 ---> Using cache
 ---> ce9f50f9568b
Step 4/20 : COPY bundled.json .
 ---> Using cache
 ---> c4bfe2cf95a5
Step 5/20 : COPY bundler.js .
 ---> Using cache
 ---> 4204bf7d78d1
Step 6/20 : COPY install-build.sh .
 ---> Using cache
 ---> b36d5dffd3b0
Step 7/20 : COPY package.json .
 ---> Using cache
 ---> 42ac9881f310
Step 8/20 : COPY src src
 ---> Using cache
 ---> c47d79beb838
Step 9/20 : RUN ./install-build.sh
 ---> Using cache
 ---> 204fdc047011
Step 10/20 : FROM httpd:2.4
 ---> c562eeace183
Step 11/20 : RUN apt-get update -y && apt-get install -y gettext
 ---> Running in 8678f45e565c
Get:2 http://deb.debian.org/debian buster InRelease [122 kB]
Get:1 http://security-cdn.debian.org/debian-security buster/updates InRelease [65.4 kB]
Get:3 http://deb.debian.org/debian buster-updates InRelease [49.3 kB]
Get:4 http://deb.debian.org/debian buster/main amd64 Packages [7907 kB]
Get:5 http://deb.debian.org/debian buster-updates/main amd64 Packages [5792 B]
Reading package lists...
E: Release file for http://security.debian.org/debian-security/dists/buster/updates/InRelease is not valid yet (invalid for another 31min 52s). Updates for this repository will not be applied.
```

sha1 of node images : 
```
docker pull node:8-alpine
8-alpine: Pulling from library/node
Digest: sha256:38f7bf07ffd72ac612ec8c829cb20ad416518dbb679768d7733c93175453f4d4
Status: Image is up to date for node:8-alpine
docker.io/library/node:8-alpine
```

On Windows, with Docker Desktop
```
$ docker version
Client: Docker Engine - Community
 Version:           19.03.5
 API version:       1.40
 Go version:        go1.12.12
 Git commit:        633a0ea838
 Built:             Wed Nov 13 07:29:52 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.5
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.12
  Git commit:       633a0ea
  Built:            Wed Nov 13 07:29:19 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

**Comment by MaxenceG2M** [comment]
Every day, I learn something.

https://askubuntu.com/questions/1059217/getting-release-is-not-valid-yet-while-updating-ubuntu-docker-container

TL;DR: Reboot your computer.
